### PR TITLE
update Hover effect on all buttons

### PR DESCRIPTION
**Pull request by anurag4499**

### CODE REVIEW

**Comment by windsonsea** [comment]
Please keep patient to wait for reviewer come in. You cannot approve any PR opened by yourself.

**Comment by utkarsh-singh1** [comment]
Hi @anurag4499, Always write a description on the PR as what changes are from your side, will help reviewers understand the PR.

File: static/css/community.css [hunk 1]
```diff
@@ -205,6 +205,11 @@ body.cid-community a.community-cta-button {
   text-align: center;
 }
 
+body.cid-community a.community-cta-button:hover {
+  background-color: #2d8ff7;
+  color: white;
+}
+
```
Comment (Gauravpadam) [inline static/css/community.css:212]: These LGTM

File: static/css/training.css [hunk 2]
```diff
@@ -178,6 +178,11 @@ body.cid-training .button {
   text-align: center;
 }
 
+body.cid-training .button:hover{
+  color: #ffffff;
+  background-color: #2f8ff5;
+}
+
```
Comment (Gauravpadam) [inline static/css/training.css:185]: These LGTM too

File: assets/scss/_base.scss [hunk 3]
```diff
@@ -77,6 +77,13 @@ footer {
     font-size: 1rem;
     border: 0px;
   }
+  
+  .button:hover {
+
+    color: white;
+    background-color:#298af5;
+
+  }
```
Comment (Gauravpadam) [inline assets/scss/_base.scss:86]: I would then put this pseudo class inside the button class above
```suggestion
    :hover {
      color: white;
      background-color:#298af5;
  }
```

**Review by Gauravpadam — CHANGES_REQUESTED** [review]
Some Suggestions from my end

**Comment by tengqm** [comment]
Looks like the improvement is done and merged in another PR, shall we close this one?

**Comment by onlydole** [comment]
Closing this due to this PR: https://github.com/kubernetes/website/pull/43175

/close

**Pull request closed without merging**